src/ConfigParser.h

```diff
@@ -202,6 +273,11 @@ class ConfigParser
         int lineNo; ///< Current line number
     };
 
+    /// parses the next ACL parameter
+    char *aclToken();
```

Comment:
The suggestion below does not address the following problem that should be addressed (possibly first): It is not clear what the difference between optionalAclToken() and this aclToken() method is.

We should be more explicit about its extraction functionality because it implies that calling this method twice does not return the same result (AFAICT).

The specific suggestion below assumes that these tokens/arguments are _optional_. Please adjust as needed.

```suggestion
    /// Extracts and returns the next ACL argument.
    /// If the current ACL directive has no more arguments, switches to the next directive and returns nil.
    /// XXX: The difference compared to optionalAclToken() is not clear.
    char *optionalAclArgument();
```